tests/wdio/spec/rsi-saverestore.spec.js

```diff
@@ -0,0 +1,251 @@
+      expect(questionnaireElementExists).to.equal(true)
+      browser.call(done)
+    })
+
+    it('Enter Data in to 0102 questionnaire', function(done) {
+      const questionnaireElementExists = browser.isExisting('.qa-questionnaire-form')
+      expect(questionnaireElementExists).to.equal(true)
+      const submitBtn = browser.element('.qa-btn-submit')
+      submitBtn.waitForExist(10000)
+      browser.setValue('[id = "94f368e4-7c6c-4272-a780-8c46328626a2-year"]', '2016')
+      browser.setValue('#dc156715-3d48-4af3-afed-7a0a6bb65583-year', '2016')
+      const saveandcontinueBtn2 = browser.element('.qa-btn-submit')
+      saveandcontinueBtn2.click();
+      const erroralert = browser.element('.alert__body')
+      const Q1text = erroralert.getText()
+      //  expect(Q1text).to.have.string('These must be corrected to continue.')
+      const experror = 'The \'to\' date must be different to the \'from\' date.'
+      if(Q1text.indexOf(experror) > -1) {
+        console.log('Expected Text is present: ' + experror );
+      }else {
+        console.log(Q1text.indexOf(experror) !== -1);
+        assert.equal(Q1text, experror);
+      }
+      browser.call(done)
+    })
+
+    it('Progress from the developer page for 1_0112', function() {
+      const userId = '.qa-user-id'
+      const collectionSID = '.qa-collection-sid'
+      const selectSchema = '.qa-select-schema'
+      const periodID = 'period_id'
+      const periodString = '.period_str'
+      browser.url('/dev')
+      browser.waitForExist(userId)
+      browser.setValue(userId, 'yoganandkunche')
+      browser.waitForExist(collectionSID)
+      browser.setValue(collectionSID, '7890112')
+      browser.waitForExist(selectSchema)
+      browser.selectByValue(selectSchema, '1_0112.json')
+      browser.click('.qa-btn-submit-dev')
+    })
+
+    it('The landing page has been reached for 0112', function(done) {
+      const getStartedBtn = browser.element('.qa-btn-get-started')
+      getStartedBtn.waitForExist(10000)
+      const url = browser.url().value
+      expect(url).to.contain('introduction')
+      getStartedBtn.click()
+      browser.call(done)
+    })
+
+    it('The questionnaire page for 0112 has been reached', function(done) {
+      const questionnaireElementExists = browser.isExisting('.qa-questionnaire-form')
+      expect(questionnaireElementExists).to.equal(true)
+      browser.call(done)
+    })
+
+    it('Enter Data in to 0112 questionnaire', function(done) {
+      const questionnaireElementExists = browser.isExisting('.qa-questionnaire-form')
+      expect(questionnaireElementExists).to.equal(true)
+      const submitBtn = browser.element('.qa-btn-submit')
+      submitBtn.waitForExist(10000)
+      browser.setValue('#fad63234-1083-4f6a-826a-20b5df6e4baa-year', '2016')
+      browser.setValue('#c8c4bd92-fd45-4fd1-83b6-18c813de2df2-year', '2017')
+      const saveandcontinueBtn2 = browser.element('.qa-btn-submit')
```

Comment:
Would be good to clean up these variable names with camelCase

`saveandcontinueBtn2` becomes `saveAndContinueBtn2`.

We can probably drop the numeric suffixes too as they are scoped to that function and so don't need to be differentiated between (unless I'm missing the reason for this)